src/main/java/org/mskcc/cmo/messaging/Gateway.java

```diff
@@ -14,6 +16,10 @@
 
     void subscribe(String subject, Class messageClass,
             MessageConsumer messageConsumer) throws Exception;
+    
+    Message request(String subject, Object message);
```

Comment:
just realized that the request and reply. methods should also have `throws Exception` like the pub/sub methods